script/Ciel/Day 14/14_01A_CIEL14A_4.txt

```diff
@@ -666,32 +720,35 @@ And yet, there isn't a single cause of death on Arcueid.
 [sha:46e440a646dd4b68891723e2111bb899d79b4142]{
 -- Page 141, Offset 38022.
 -- あれはこの自然界の上にいるだけで完璧な生命だと、前に誰かが言っていた。
-There was someone who once told me that she was a perfect lifeform only while she stands on top of this realm of nature.
+// They did? I can't find any other mentions of 自然界 or 完璧な生命 in Ciel route other than D14
+Someone once told me that so long as she exists within nature, she is a perfect lifeform.
```

Comment:
Yes, in original Tsukihime!
Nasu be copy-pasting too hard.
Add this as a comment just because it's a funny fact
>問題は、彼らの吸血衝動に理由がない、という事なんです。
　理由がないから止めようがない。
　真祖という完璧な生命が内包した欠点。死に至る病、とでも言いましょうか。
　とにかく彼らは自らの“血を吸いたい”という衝動を抑えて生きています。